### PR TITLE
adding pom file in teja-dev1.0

### PR DESCRIPTION
**Pull request by ganesh4845**
Please review pom.xml file

### CODE REVIEW

File: maven-pom.xml.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.openjfx</groupId>
+  <artifactId>hellofx</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0-SNAPSHOT</version>
```
Comment (hardevops) [inline maven-pom.xml.txt:7]: Please change the version 1.0 to 1.1

File: maven-pom.xml.txt [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <javafx.version>21</javafx.version>
+    <javafx.maven.plugin.version>0.0.8</javafx.maven.plugin.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.openjfx</groupId>
```
Comment (hardevops) [inline maven-pom.xml.txt:19]: give groupid as jdk1.8

**Review by hardevops — CHANGES_REQUESTED** [review]
please work on review comments.

File: maven-pom.xml.txt (hunk 2 above)
Comment (ganesh4845) [reply maven-pom.xml.txt:19]: added jdk 1.8

**Review by hardevops — APPROVED** [review]
looks good.

**Pull request merged**